data-prepper-plugins/csv-processor/src/main/java/org/opensearch/dataprepper/plugins/codec/csv/CsvOutputCodecConfig.java

```diff
@@ -17,11 +19,56 @@ public class CsvOutputCodecConfig {
     @JsonProperty("header")
     private List<String> header;
 
+    @JsonProperty("exclude_keys")
+    private List<String> excludeKeys;
+    @Valid
+    @Size(max = 0, message = "Header from file is not supported.")
+    @JsonProperty("header_file_location")
+    private String headerFileLocation;
+
+    @Valid
+    @Size(max = 0, message = "Header from file is not supported.")
+    @JsonProperty("region")
+    private String region;
+    @Valid
+    @Size(max = 0, message = "Header from file is not supported.")
+    @JsonProperty("bucket_name")
+    private String bucketName;
+    @Valid
+    @Size(max = 0, message = "Header from file is not supported.")
```

Comment:
This error message is copied from `header_file_location`

Reply:
Merging this for now. Please fix this @umayr-codes

Reply:
These error messages were written in accordance with Ashok's suggestion!